app/views/cases/shared/_case_deadlines.html.slim

```diff
@@ -4,6 +4,10 @@
     td = case_details.internal_deadline
 
 - if case_details.external_deadline.present?
-  tr.section.case-external-deadline
+  tr.section.case-external-deadline.section-external-deadline
     th = t('common.case.external_deadline')
     td = case_details.external_deadline
+    / note this link is to edit the date responded date after a case is closed 
+    / placed here due to where it needs to sit on page
+    - if case_details.closed? 
+      td = link_to t('common.links.change'), edit_step_case_sar_offender_path(case_details, "date_responded")
```

Comment:
I think we might be printing this out on closed cases from other correspondence types as a result of this 🤔 

And it's a bit weird having the edit link in the deadlines partial as your comment explains

In another PR we restructured the case details sections so that the partials more closely aligned to the steps (e.g. subject details) - I wonder if we could make a new partial for offender sar displaying the fields in this area of the case details partial?

Reply:
Yeah that is a good point I think it would be useful to do this, it feels wrong doing it this way.